Sen6x

mebner86 opened the pull request:
## Description:

**Related issue (if applicable):** fixes [issue 2995](https://github.com/esphome/feature-requests/issues/2995)

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** 

- [esphome/esphome#12553](https://github.com/esphome/esphome/pull/12553)

## Checklist:

  - [x] I am merging into `next` because this is new documentation that has a matching pull-request in [esphome](https://github.com/esphome/esphome) as linked above.  
    or
  - [ ] I am merging into `current` because this is a fix, change and/or adjustment in the current documentation and is not for a new component or feature.

  - [x] Link added in `/components/_index.md` when creating new documents for new components or cookbook.

<details>
<summary><strong>New Component Images</strong></summary>

If you are adding a new component to ESPHome, you can automatically generate a standardized black and white component name image for the documentation.

**To generate a component image:**

1. Comment on this pull request with the following command, replacing `component_name` with your component name in **lower_case** format with **underscores** (e.g., `bme280`, `sht3x`, `dallas_temp`):

   ```
   @esphomebot generate image component_name
   ```

2. The ESPHome bot will respond with a downloadable ZIP file containing the SVG image.

3. Extract the SVG file and place it in the `/static/images/` folder of this repository.

4. Use the image in your component's index table entry in `/components/_index.md`.

**Example:** For a component called "DHT22 Temperature Sensor", use:
```
@esphomebot generate image dht22
```

</details>

piitaya commented on content/components/sensor/sen6x.md:
```diff
@@ -0,0 +1,225 @@
+
+  - All other options from [Sensor](/components/sensor).
+
+- **nox** (*Optional*): NOx Index. Note: Only available with Sen55. The sensor will be ignored on unsupported models.
```
```suggestion
- **nox** (*Optional*): NOx Index. Note: Only available with Sen65, Sen66, Sen66 or Sen69C. The sensor will be ignored on unsupported models.
```

Commit:
[sen6x] Add SEN6x sensor documentation

Add documentation for the Sensirion SEN6x series environmental sensor
(SEN62, SEN63C, SEN65, SEN66, SEN68, SEN69C), covering basic
functionality as implemented in esphome/esphome#12553. Add
cross-reference from sen5x page.

Commit:
[sen6x] Add SEN6x to components index table

Commit:
[sen6x] Scale product image to 70% and add alt texts

swoboda1337 committed:
Fix

swoboda1337 committed:
Fix

swoboda1337 committed:
Fix

swoboda1337 committed:
Fix

swoboda1337 committed:
Add image

swoboda1337 committed:
Fix

swoboda1337 committed:
Clean
